server/routes/reporting/standard-report/definitions.js

```diff
@@ -1119,6 +1151,7 @@
           'courtName',
           'totalOnCall',
         ],
+        defaultSortColumn: 'lastName',
       },
       'trial-attendance': {
```

Comment:
BespokeReport.insertRows function seems to be breaking the sorting on the report while viewing, maybe make this unsortable or we need to stop these inserted rows from being sorted as they should always be at bottom of each table

Reply:
I missed one 😄 yes, we need to find a good way of injecting an attribute of `fixed-row-index` to block it from being sorted... or I can try and do something clever with the sorter.